push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -61,7 +75,7 @@ public void sendMessagesToPushNetwork(@Observes @Dequeue MessageHolderWithTokens
         final UnifiedPushMessage unifiedPushMessage = msg.getUnifiedPushMessage();
         final Collection<String> deviceTokens = msg.getDeviceTokens();
 
-        logger.info(String.format("Received UnifiedPushMessage from JMS queue, will now trigger the Push Notification delivery for the %s variant (%s)", variant.getType().getTypeName(), variant.getVariantID()));
+        logger.info("Received UnifiedPushMessage from JMS queue, will now trigger the Push Notification delivery for the {} variant {}", variant.getType().getTypeName(), variant.getVariantID());
 
```

Comment:
Can you update this statement? In the notification dispatcher ticket we had:

`UnifiedPushMessage was successfully received. Push Notification delivery for the {} variant ({}) will now be triggered.`